src/helpers/LaunchCamera.tsx

```diff
@@ -42,7 +43,13 @@ const takePicture = async (
           setResponse,
         );
       } else {
-        Alert.alert('Camera permission denied');
+        Toast.show({
+          type: 'error',
+          text1: 'Permission denied',
```

Comment:
1. give text1 and text2 to some meaningful name.
2. Move 'Permission denied' to en.ts file.

Reply:
text 1 text2 are properties provided by the package only we can't change that

Reply:
aah ok my bad